tests/Integration/Behaviour/Features/Context/Domain/LanguageFeatureContext.php

```diff
@@ -0,0 +1,339 @@
+            }
+        } catch (LanguageException $e) {
+            $this->setLastException($e);
+        }
+    }
+
+    /**
+     * @When /^I (enable|disable) the language with ISOCode "([a-z]{2})"$/
+     *
+     * @param string $statusVerb
+     * @param string $isoCode
+     */
+    public function setStatusLanguage(string $statusVerb, string $isoCode): void
+    {
+        try {
+            $this->cleanLastException();
+            $this->getCommandBus()->handle(
+                new ToggleLanguageStatusCommand(
+                    SharedStorage::getStorage()->get($isoCode),
+                    $statusVerb === 'enable'
+                )
+            );
+        } catch (LanguageException $e) {
+            $this->setLastException($e);
+        }
+    }
+
+    /**
+     * @When /^I bulk (enable|disable) languages with ISOCode "([a-z,]+)"$/
+     *
+     * @param string $statusVerb
+     * @param string $isoCodes
+     */
+    public function bulkSetStatusLanguage(string $statusVerb, string $isoCodes): void
+    {
+        $isoCodes = explode(',', $isoCodes);
+        $languageIds = [];
+        foreach ($isoCodes as $isoCode) {
+            $languageIds[] = SharedStorage::getStorage()->get($isoCode);
+        }
+
+        try {
+            $this->cleanLastException();
+            $this->getCommandBus()->handle(
+                new BulkToggleLanguagesStatusCommand($languageIds, $statusVerb === 'enable')
+            );
+        } catch (LanguageException $e) {
+            $this->setLastException($e);
+        }
+    }
+
+    /**
+     * @Then the language with ISOCode :isoCode should have the following details:
+     *
+     * @param string $isoCode
+     * @param TableNode $table
+     */
+    public function checkLanguageDetails(string $isoCode, TableNode $table): void
+    {
+        $editableLanguage = $this->getLanguage($isoCode);
+
+        $this->assertLastErrorIsNull();
+
+        $data = $table->getRowsHash();
+        Assert::assertEquals($data['name'], $editableLanguage->getName());
+        Assert::assertEquals($data['isoCode'], $editableLanguage->getIsoCode()->getValue());
+        Assert::assertEquals($data['tagIETF'], $editableLanguage->getTagIETF()->getValue());
+        Assert::assertEquals($data['shortDateFormat'], $editableLanguage->getShortDateFormat());
+        Assert::assertEquals($data['fullDateFormat'], $editableLanguage->getFullDateFormat());
+        Assert::assertEquals((bool) $data['isRtl'], $editableLanguage->isRtl());
+        Assert::assertEquals((bool) $data['isActive'], $editableLanguage->isActive());
+        Assert::assertEquals([
+            SharedStorage::getStorage()->get($data['shop']),
+        ], $editableLanguage->getShopAssociation());
+    }
+
+    /**
+     * @Then the language with ISOCode :isoCode should exist
+     *
+     * @param string $isoCode
+     */
+    public function checkLanguageExists(string $isoCode): void
+    {
+        $this->getLanguage($isoCode);
```

Comment:
```suggestion
        $this->assertNull($this->getLanguage($isoCode));
```

Just a suggestion, the exception alone is enough to validate the behaviour